src/layouts/MetricsLayout.tsx

```diff
@@ -0,0 +1,37 @@
+import type { ReactNode, PropsWithChildren } from "react";
+import { useAccount } from "wagmi";
+
+import Header from "~/components/Header";
+import BallotOverview from "~/features/ballot/components/BallotOverview";
+import { BaseLayout, type LayoutProps } from "./BaseLayout";
+import {
+  useCurrentDomain,
+  useCurrentRound,
+} from "~/features/rounds/hooks/useRound";
+import { useRoundState } from "~/features/rounds/hooks/useRoundState";
+import { useSession } from "next-auth/react";
+import { Button } from "~/components/ui/Button";
+import Link from "next/link";
+
+type Props = PropsWithChildren<
+  {
+    sidebar?: "left" | "right";
+    sidebarComponent?: ReactNode;
+  } & LayoutProps
+>;
+export const MetricsLayout = ({ children, ...props }: Props) => {
```

Comment:
The layout components might be a good candidate for refactoring in the future.

Reply:
Noted, yes I just used it to change a bit the width, according to how the other pages are used, but we will change it